doc/readthedocs/sphinx/auxiliary-programs/auxiliary-programs.rst

```diff
@@ -938,7 +938,7 @@ following three tables illustrate the names for data elements.
   |                | rmal_radiation |                |                |
   +----------------+----------------+----------------+----------------+
   | horirsky       | horizon        | W              | Y              |
-  |                | tal_infrared\_ | h m\ :sup:`−2` |                |
+  |                | tal_infrared_  | h m\ :sup:`−2` |                |
```

Comment:
I think this table still needs love.  Would you want to give this another pass?  https://energyplus.readthedocs.io/en/10605documentationissues2024b/auxiliary-programs/auxiliary-programs.html#id68

If not, I can.  The table is really just too skinny.  I would try to use a tool like https://www.tablesgenerator.com/text_tables to paste in the data and have it generate a nice fully formed version.

Reply:
I can look at this further and fully clean this table up.